services/app/src/actions/User.tsx

```diff
@@ -182,10 +184,36 @@ function updateState(dispatch: Redux.Dispatch<any>): ((u: UserState) => Promise<
   };
 }
 
+function fetchUserFeedbacks(): Promise<IUserFeedback[]> {
+  return fetch(AUTH_SETTINGS.URL_BASE + '/user/feedbacks', {
+    credentials: 'include',
+    headers: {
+      'Content-Type': 'text/plain',
+    },
+    method: 'GET',
+  })
+  .then(handleFetchErrors)
+  .then((response: Response) => response.json())
+  .then((feedbacks: IUserFeedback[]) => feedbacks);
+}
+
+type TReduxThunk<ReturnType> = (dispatch: Redux.Dispatch<any>, getState: () => AppState) => ReturnType;
+
+export function logoutUser(): TReduxThunk<Promise<void>> {
+  return (dispatch) => {
+    const gapi = getGapi();
+    const auth2 = gapi.auth2.getAuthInstance();
+    return auth2.signOut().then(() => {
+      dispatch({type: 'USER_LOGOUT'});
+      dispatch(toCard({name: 'TUTORIAL_QUESTS'}));
```

Comment:
Seeing this, realizing that it would be good to, after changing cards, also show a snackbar saying "Logged out successfully", so that the user is absolutely certain that they were logged out and not just redirected to another page :)